### PR TITLE
DOC: doc warnings

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

File: doc/source/whatsnew/v0.10.1.txt [hunk 1]
```diff
@@ -58,7 +58,8 @@ perform queries on a table, by passing a list to ``data_columns``
 
    # on-disk operations
    store.append('df', df, data_columns = ['B','C','string','string2'])
-   store.select('df',[ 'B > 0', 'string == foo' ])
+   store.select('df').dtypes
```
Comment (jreback) [inline doc/source/whatsnew/v0.10.1.txt]: @jorisvandenbossche attempting to see what is going on here......

**Comment by jreback** [comment]
don't merge yet.

**Comment by jorisvandenbossche** [comment]
@jreback Fixed the problems with #15620 in that PR, to keep the credit.

**Comment by jreback** [comment]
ok thanks

**Comment by jreback** [comment]
ok I think this fixed the HDFStore issues.

**Comment by jorisvandenbossche** [comment]
Cool. 
Is the list with with expressions not supported anymore? Or just for some reason failing on travis?

**Comment by jreback** [comment]
@jorisvandenbossche no a list is still accepted (but nice with the and). The issue was the bare strings, e.g. ``bar=foo``; for some reason the foo is interpreted as an ascii string I think, you need to use the correct quotes ``"``. I cannot repro this so maybe its an encoding issue (IOW even though we are building with 3.5, maybe the encoding of the file itself is ascii, just guessing here).

I put a note on the original issue #15544 to see if @chris-b1 has any ideas.

**Comment by jreback** [comment]
@jorisvandenbossche ok this fixes a bunch of warnings (still a couple remaining) :>

some png links and a LICENSE link........but enough for now.

**Comment by jorisvandenbossche** [comment]
The png and license link warnings are already present for a few years, so not too urgent :-)

**Comment by chris-b1** [comment]
The issue here was that `foo` was a defined name in a different whatsnew [section](http://pandas-docs.github.io/pandas-docs-travis/whatsnew.html#whatsnew-0170-enhancements-other)
```python
foo = pd.Series([1,2], name='foo')
```

Now this comparison was unpacking the ints in that `Series` (which tbh I didn't we did - but basically is like `.isin`) and raising the now invalid comparison.  
```python
store.select('df',[ 'B > 0', 'string == foo' ])
```

In past doc versions, this was just returning an empty result ([here](http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#hdfstore)).  This is exactly that type of invalid comparisons we were trying to catch with #15544, so I think it's good that this raised!

**Comment by jreback** [comment]
@chris-b1 ahh that makes sense then. thanks for the look!

**Pull request closed without merging**